sdk/cmake/msvc.cmake

```diff
@@ -164,7 +164,9 @@ if(MSVC_IDE AND (NOT DEFINED USE_FOLDER_STRUCTURE))
     set(USE_FOLDER_STRUCTURE TRUE)
 endif()
 
-if(RUNTIME_CHECKS)
+if (SANITIZE_UB)
+    #add_compile_options(/fsanitize=address)
```

Comment:
Doesn't seem to be useful. MSVC doesn't also have this.

Reply:
Viktor added this, so I will leave as is in my branch. Can fix it if he does though